kong/router/atc.lua

```diff
@@ -423,6 +387,10 @@ do
 end
 
 
+local MATCH_PARAMS = {}
```

Comment:
Use https://github.com/Kong/kong/blob/master/kong/tools/request_aware_table.lua?

Reply:
I think that module level table is OK here.